README.md

```diff
@@ -130,10 +206,111 @@ jenkins2_credentials:
     description: 'username and password for jira'
     username: 'jiraUser'
     password: 'JiraUserPassword'
+  pipeline_libraries:
+    type: 'password'
+    id: 'pipeline_libraries'
+    description: 'username and password for pipeline libraries'
+    username: 'pipelineLibrariesUser'
+    password: 'pipelineLibrariesPassword'
+  sshconnection:
+    type: 'key'
+    keySource: 0
+    key: >
+      -----BEGIN RSA PRIVATE KEY-----
```

Comment:
Should we store ssh private key in public?

Reply:
it is just as an example